packages/appium/lib/logsink.js

```diff
@@ -186,8 +186,26 @@ async function createTransports(args) {
   return transports;
 }
 
+let encounteredPrefixes = [];
```

Comment:
let's use `const`

Reply:
Will that still let me append?

Reply:
(i.e., `const` doesn't mean "make this thing immutable", it just means "don't let me reassign this name")

Reply:
Ahhh! I see!

Reply:
Changed to const and moved to top